introduce patch(.diff) to slstatus and dmenu

ahdyt opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

ahdyt committed:
introduce patch(.diff) to slstatus and dmenu

fnlaai commented:
@oxzi Alv, I'm introduce patches, I already tested with sudo nixos-rebuild switch -I nixpkgs=/cloned/github/nixpkgs, and it works.

NobbZ commented on pkgs/top-level/all-packages.nix:
```diff
@@ -20428,8 +20429,12 @@ in
 
-  dmenu = callPackage ../applications/misc/dmenu { };
-  dmenu-wayland = callPackage ../applications/misc/dmenu/wayland.nix { };
+  dmenu = callPackage ../applications/misc/dmenu {
```
The `dmenu` package usually has a default of `[ ./xim.patch ]` in it, you are basically removing it here.

Is this intentional?

oxzi replied:
This old default patch was removed by bumping to 5.0 in #98267. Now the same `patches` code can be used as for `slstatus` above.

NobbZ commented on pkgs/top-level/all-packages.nix:
```diff
@@ -20428,8 +20429,12 @@ in
+  dmenu = callPackage ../applications/misc/dmenu {
+    patches = config.dmenu.patches or [];
+  };
+  dmenu-wayland = callPackage ../applications/misc/dmenu/wayland.nix {
```
The function returned from that file doesn't even have `patches` in it's argset, are you sure this works?

I haven't tested it though, as I'm doing the comments from a mobile.

NobbZ replied:
Just see it in the error of the GitHub action...

It does indeed not evaluate because of this:

https://gist.github.com/GrahamcOfBorg/0f56c0ebd42091febfe5410583975bf6#file-output-path-comparison-L9

NobbZ reviewed (commented):
On top of the 2 comments I have left inline, I do not like that the config from the module system are used like this, I think instead the module system should use `override`/`overrideAttrs` to apply patches and changes.

Though in the chat I was told that this PR is based on prior art with other "suckless" software packages.

As this review is solely based on personal opinion and reading the diff/referred files and not actually running any builds that would verify buildability or functionality of these changes, this is posted as "comment"

fnlaai commented:
I'm done here, although I can build derivation on my own, so get lost those nix newcomer who will follow pattern nixpkgs.config.<suckless's-packages>.patches ! make your own overlays/derivation~

NobbZ commented:
Sorry if I offended you, that was not my intend.

I was just asking questions, about how you do things, and if you realized how your changes might affect others, as you overwrite current default behaviour with the patch for `dmenu` beeing basically "deleted".

Also I pointed out that another of your changes does not build at all.

Last but not least, I realise that your work is based on prior art and because of that has to be considered "okay", even though I do not agree with the prior art.

If you want to, you can totally ignore my comment about not liking the way how the changes use the modules, but still, the breaking build of `dmenu-wayland` remains and my question about your intend of dropping the `./xim.patch` for `dmenu`.

I'm totally fine with the intend of making those software easier to configure via modules, though not at the cost of failing builds or unintentionally dropped patches.

So please fix the build for `dmenu-wayland`, or the PR can't be merged, as its not even passing simple evaluation checks. This is the bare minimum to get a proper and meaningful review of someone more experienced with the suckless ecosystem and their integration into nixos/nixpkgs.

ahdyt committed:
fixing dmenu-wayland not taking patches arg

fnlaai commented:
No, it's my fault, I have no idea what xim patch do, and the wayland one, thought it was take patches args, ah.

NobbZ commented:
I think we should ping for a review:

* @oxzi as the maintainer of `slstatus`
* @pSub as the maintainer of `dmenu`

ahdyt committed:
Merge branch 'master' of https://github.com/nixos/nixpkgs into master

oxzi commented on pkgs/applications/misc/dmenu/default.nix:
```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, libX11, libXinerama, libXft, zlib, patches ? [ ./xim.patch ] }:
```
This became obsolete after #98267 landed. Thus, the PR should be rebased.

oxzi commented on pkgs/top-level/all-packages.nix:
```diff
@@ -7165,6 +7165,7 @@ in
   slsnif = callPackage ../tools/misc/slsnif { };
 
   slstatus = callPackage ../applications/misc/slstatus {
+    patches = config.slstatus.patches or [];
```
Looks good to me.

oxzi reviewed (changes requested):
Please excuse my late response, @h1dyt. Everything looks good to me, except the legacy `dmenu` changes which became obsolete after #98267. Could you please rebase your PR? Thanks!

ahdyt committed:
Merge branch 'master' of https://github.com/nixos/nixpkgs into master

h1dyt commented:
I've updated my repo but it's kinda adding new "commits" ? is that how to rebase? I guess you've should to squash it.

ahdyt committed:
remove xim.patch opts dmenu from top-level

oxzi commented on pkgs/top-level/all-packages.nix:
```diff
@@ -20651,8 +20652,10 @@ in
-  dmenu = callPackage ../applications/misc/dmenu { };
-  dmenu-wayland = callPackage ../applications/misc/dmenu/wayland.nix { };
+  dmenu = callPackage ../applications/misc/dmenu {
+    patches = config.dmenu.patches or [ ../applications/misc/dmenu/xim.patch ];
```
The `xim` patch became obsolete. Thus, you should change this to an empty list.

oxzi reviewed (changes requested):
Sorry, I missed the faulty default in the `all-packages.nix` before. Could you please change this one again? Thanks a lot!

ahdyt reviewed (commented):
> Sorry, I missed the faulty default in the `all-packages.nix` before. Could you please change this one again? Thanks a lot!

yes, ofborg reminds me.

oxzi reviewed (approved):
Looks good to me (:

ahdyt committed:
update from nixos

* perlPackages.LaTeXML: enable ImageMagick

* jamulus: 3.6.2 -> 3.7.0

* eiskaltdcpp: remove build dependency from boost (#116998)

eiskaltdcpp: remove build dependency from boost

+ Remove eiskaltdcpp-cli-xmlrpc: it is absolutely useless because XML-RPC
  support in eiskaltdcpp-daemon is currently in a broken state
+ Update cmake options
+ eiskaltdcpp: install eiskaltdcpp-cli-jsonrpc
+ eiskaltdcpp: symlink $pname-qt to $pname

* digitemp: init at 3.7.2

This does not include digitemp_DS2490, as that seems to require libusb0.1,
which isn't in nixpkgs.

* metasploit: 6.0.34 -> 6.0.36

* nco: 4.9.7 -> 4.9.8 + fix license and deps

* home-manager: 2021-01-16 -> 2021-03-21

* elmPackages.elm-format: fix build

closes #117336

* jellyfin: 10.7.0 -> 10.7.1

* jmol: 14.31.34 -> 14.31.35

* rpm2targz: don't put runtime deps into buildInputs

* zs-wait4host: don't put runtime deps into buildInputs

* zstxtns-utils: don't put runtime deps into buildInputs

* python3Packages.tatsu: 5.6.0 -> 5.6.1

* vscode-extensions.redhat.java: mark as broken, not available

Is available for jdk >= 11 and broken for jdk < 11.

* poke: 1.0 -> 1.1

* linux: 4.19.181 -> 4.19.182

* linux: 5.10.24 -> 5.10.25

* linux: 5.11.7 -> 5.11.8

* linux: 5.4.106 -> 5.4.107

* linux-rt_5_4: 5.4.102-rt53 -> 5.4.106-rt54

* linux/hardened/patches/4.19: 4.19.181-hardened1 -> 4.19.182-hardened1

* linux/hardened/patches/5.10: 5.10.24-hardened1 -> 5.10.25-hardened1

* linux/hardened/patches/5.11: 5.11.7-hardened1 -> 5.11.8-hardened1

* linux/hardened/patches/5.4: 5.4.106-hardened1 -> 5.4.107-hardened1

* python3Packages.sagemaker: 2.30.0 -> 2.31.0

* kodiPackages.pvr-hts: 8.2.2 -> 8.2.4

* rust-analyzer: 2021-03-15 -> 2021-03-22

* fuse: use macfuse-stubs instead on Darwin

* drone-runner-docker: init at 1.6.3 (#117239)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

* dockerTools.buildLayeredImage: support fromImage

It is now possible to pass a `fromImage` to `buildLayeredImage` and
`streamLayeredImage`, similar to what `buildImage` currently supports.

This will prepend the layers of the given base image to the resulting
image, while ensuring that at most `maxLayers` are used. It will also
ensure that environment variables from the base image are propagated
to the final image.

* python3Packages.sphinx-navtree: mark as broken

* python3Packages.sklearn-deap: mark as broken

* perlPackages.LaTeXML: move makeWrapper to nativeBuildInputs

* musl: 1.2.1 -> 1.2.2

https://git.musl-libc.org/cgit/musl/tree/WHATSNEW?h=v1.2.2#n2242

* multimc: unstable -> 0.6.12

* python3Packages.rabbitpy: mark as broken

* tz: init at 0.4

* doc: add section for FUSE

* pspg: 4.3.1 -> 4.4.0

* notcurses: 2.2.2 -> 2.2.3

* links2: 2.21 -> 2.22

* python38Packages.azure-multiapi-storage: 0.6.0 -> 0.6.2

* mksh: fix license

According to official documentation (http://www.mirbsd.org/TaC-mksh.txt), mksh
licensing changes according to build-time configuration.

* python38Packages.bitarray: 1.7.1 -> 1.8.0

* salt: 3002.5 -> 3002.6

* zchunk: 1.1.8 -> 1.1.9

* ldeep: 1.0.9 -> 1.0.10

* qt5: Include the "out" derivation as well as the "dev" in qt5.full

Without a number of important Qt libraries (all of those with a "dev"
derivation) are not included in the bundled package, and qmake can't
find them.

Fixes #110199

* josm: 17560 -> 17580

* python3Packages.restview: 2.9.1 -> 2.9.2 (#117374)

* zs-apc-spdu-ctl: don't put runtime deps into buildInputs

* zziplib: 0.13.71 -> 0.13.72

* gitea: 1.13.5 -> 1.13.6

- minor bugfix release including security fixes, see https://github.com/go-gitea/gitea/releases/tag/v1.13.6

* gitea: normalise derivation format using `nixpkgs-fmt`

* sipvicious: init at 0.3.2

* stdenv: Fix regression on ARM+static when enabling hardening (#115363)

4e9dc46dea0ef8cf15c567fa863796bb23099d0b re-enabled hardening for Musl,
which is good.

Though static builds for ARM fail in various ways

 - cross armv7l static does not build
 - cross aarch64 static produces segfaulting dynamically linked binaries
 - native aarch64 static also produces segfaulting dynamically linked binaries

It seems that for native x86_64-linux, static builds are fine though.

This works around the issue by removing PIE from the hardening flags,
keeping all other hardening flags. This is an improvement (I think) from
before 4e9dc46d.

Fixes #114953

* python3Packages.loguru: unbreak (#117381)

* vimPlugins: update

* vimPlugins.nvim-bufferline-lua: init at 2021-03-22

* python3Packages.forbiddenfruit: unbreak

* udunits: 2.2.27.6 -> unstable-2021-03-17

* plasma-workspace: fix path to qdbus

* pythonPackages.pyradios: init at 0.0.22

* python3Packages.pymumble: 0.3.1 -> 1.6

* botamusique: init at unstable-2021-03-13

* bit: init at 1.05

* weechat-notify-send: fixed meta.homepage mispelling

* cypari2: 2.1.1 -> 2.1.2

* cypari2: 2.1.1 -> 2.1.2, patch to fix segfault

* sage: update test expectations

* sage: patch intermittently failing test

* cysignals: 1.10.2 -> 1.10.3

* gap: backport aarch64 patch

* discord: 0.0.13 -> 0.0.14

* cargo-audit: add cargo audit fix by default

* libpst: build as shared library

* evolution: enable importing Outlook .pst data files

* rubocop: 1.11.0 -> 1.12.0

https://github.com/rubocop/rubocop/releases/tag/v1.12.0

* python3Packages.plexapi: 4.4.1 -> 4.5.0

* vimPlugins.nvim-colorizer-lua: init at 2020-06-11

* python38Packages.croniter: 0.3.37 -> 1.0.9

* python38Packages.azure-eventgrid: 4.0.0 -> 4.1.0

* python38Packages.mautrix: 0.8.16 -> 0.8.17

* joker: 0.15.7 -> 0.16.0

* python38Packages.databricks-connect: 7.3.10 -> 8.1.0

* buildkite-agent: 3.27.0 -> 3.28.1

* openmpt123: 0.5.5 -> 0.5.6

* python38Packages.xdis: 5.0.5 -> 5.0.8 (#117364)

* python3Packages.croniter: use pytestCheckHook

* pythonPackages.android-backup: init at 0.2.0

* python3Packages.python-miio: 0.5.4 -> 0.5.5.1

* python38Packages.sasmodels: 1.0.2 -> 1.0.4

* pyload: drop

* plowshare: spidermonkey_38 -> spidermonkey_78

* jsawk: spidermonkey_38 -> spidermonkey_78

the examples from the readme work just fine, with this version

* spidermonkey_38: drop

it was based on a firefox release that last received an update in April 2016

* helio-workstation: 3.3 -> 3.4

* nixos/wireguard: don't use alias

* mill: 0.9.3 -> 0.9.5

* pika-backup: 0.2.1 -> 0.2.2

* github-backup: init at 0.39.0 (#116976)

* doc: improve Python section (#116344)

Avoid confusion as in https://github.com/NixOS/nixpkgs/pull/116325#discussion_r593946638.

* cen64: unstable-2020-02-20 -> unstable-2021-03-12

* slowhttptest: init at 1.8.2

* chromiumDev: 91.0.4449.6 -> 91.0.4455.2

* coursier: 2.0.15 -> 2.0.16

* python3Packages.botocore: 1.20.33 -> 1.20.34

* python3Packages.boto3: 1.17.33 -> 1.17.34

* awscli: 1.19.33 -> 1.19.34

* linux-hardened: Switch to 5.11

* linux-hardened: Fix page poisoning for 5.11

`PAGE_POISONING_NO_SANITY` was removed in
https://git.kernel.org/linus/8f424750baaafcef229791882e879da01c9473b5

`PAGE_POISONING_ZERO` was removed in
https://git.kernel.org/linus/f289041ed4cf9a3f6e8a32068fef9ffb2acc5662

* python3Packages.aiosmb: init at 0.2.37

* python3Packages.pypykatz: 0.4.3 -> 0.4.7

* ghostwriter: 2.0.0-rc4 -> 2.0.0-rc5

* vscode: fix download link

* jbang: 0.68.0 -> 0.69.1

* gitAndTools.git-machete: 3.0.0 -> 3.1.0

###### Motivation for this change
Update to latest upstream version

###### Things done
 * [ ]  Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
 * Built on platform(s)

   * [x]  NixOS
   * [ ]  macOS
   * [ ]  other Linux distributions
 * [ ]  Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
 * [ ]  Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
 * [x]  Tested execution of all binary files (usually in `./result/bin/`)
 * [ ]  Determined the impact on package closure size (by running `nix path-info -S` before and after)
 * [ ]  Ensured that relevant documentation is up to date
 * [x]  Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers
cc @blitz @Ma27 @tfc @worldofpeace

* gnomeExtensions.sound-output-device-chooser: 32 -> 35

Refresh paths patch.

* python3Packages.hdbscan: unbreak

* nixos/mastodon: enable sandbox mode

* minio-client: 2021-03-12T03-36-59Z -> 2021-03-23T05-46-11Z

* python3Packages.prance: unbreak

* python3Packages.numtraits: remove

This package has long been replaced by https://github.com/jupyter-widgets/traittypes

* python3Packages.onnx: fix build

* sphinx-autobuild: Use correct package

* sphinx-autoload: Fix typo in maintainers

* nuclei: 2.3.1 -> 2.3.2

* go-task: rename executable to task (#117455)

* python3Packages.cherrypy: unbreak

* terragrunt: 0.28.15 -> 0.28.16 (#117493)

* python3Packages.awkward0: 0.15.2 -> 0.15.5 (#117397)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>
Co-authored-by: Dmitry Kalinkin <dmitry.kalinkin@gmail.com>

* python3Packages.piep: disable

* bumpver: 2020.1108 -> 2021.1110

* terraform-providers.yandex: 0.40.0 -> 0.54.0

This terraform module has been moved to the yandex-cloud organization
and the old one archived, thus not receiving any updates.

* nixUnstable: add util-linuxMinimal to nativeBuildInputs

Fixes an issue in tests where they can fail with `../common.sh: line 92:
unshare: command not found`.

* nixUnstable: skip flakey tests/ca/substitute.sh

* rtsp-simple-server: 0.15.0 -> 0.15.1

* prs: 0.2.4 -> 0.2.5

* youtube-dl: 2021.03.14 -> 2021.03.25

https://github.com/ytdl-org/youtube-dl/releases/tag/2021.03.25

* zziplib: fix missing links on darwin

* terraform-docs: 0.11.2 -> 0.12.0 (#117525)

* terraform_0_14: 0.14.8 -> 0.14.9 (#117522)

* less.nix: fix escape

$$ does not escape $
\$ does

* python3Packages.solaredge: init at 0.0.4

* home-assistant: update component-packages

* home-assistant: enable solaredge tests

* python38Packages.pyxnat: unbreak

* release notes: typo fixes (#117539)

* ajour: 0.7.2 -> 1.0.0

Co-authored-by: Vincenzo Mantova <xworld21@users.sf.net>
Co-authored-by: R. RyanTM <ryantm-bot@ryantm.com>
Co-authored-by: Boris Pek <tehnick-8@yandex.ru>
Co-authored-by: Profpatsch <mail@profpatsch.de>
Co-authored-by: zseri <zseri.devel@ytrizja.de>
Co-authored-by: Fabian Affolter <mail@fabian-affolter.ch>
Co-authored-by: stigtsp <stig@stig.io>
Co-authored-by: Bruno Bzeznik <Bruno.Bzeznik@imag.fr>
Co-authored-by: Peter Hoeg <peter@hoeg.com>
Co-authored-by: Marek Fajkus <marek.faj@gmail.com>
Co-authored-by: Maximilian Bosch <maximilian@mbosch.me>
Co-authored-by: Michael Weiss <dev.primeos@gmail.com>
Co-authored-by: sternenseemann <0rpkxez4ksa01gb3typccl0i@systemli.org>
Co-authored-by: Kira Bruneau <kira.bruneau@pm.me>
Co-authored-by: Domen Kožar <domen@dev.si>
Co-authored-by: Tim Steinbach <tim@nequissimus.com>
Co-authored-by: Timo Kaufmann <timokau@zoho.com>
Co-authored-by: Ryan Mulligan <ryan@ryantm.com>
Co-authored-by: Oleksii Filonenko <brightone@protonmail.com>
Co-authored-by: oxalica <oxalicc@pm.me>
Co-authored-by: Sandro <sandro.jaeckel@gmail.com>
Co-authored-by: midchildan <git@midchildan.org>
Co-authored-by: Danielle <dani@builds.terrible.systems>
Co-authored-by: Louis Blin <45168934+lbpdt@users.noreply.github.com>
Co-authored-by: Martin Weinelt <mweinelt@users.noreply.github.com>
Co-authored-by: Robert T. McGibbon <rmcgibbo@gmail.com>
Co-authored-by: Dmitry Kalinkin <dmitry.kalinkin@gmail.com>
Co-authored-by: TredwellGit <tredwell@tutanota.com>
Co-authored-by: Louis Bettens <louis@bettens.info>
Co-authored-by: Ben Siraphob <bensiraphob@gmail.com>
Co-authored-by: Austin Seipp <aseipp@pobox.com>
Co-authored-by: AndersonTorres <torres.anderson.85@protonmail.com>
Co-authored-by: Flakebi <flakebi@t-online.de>
Co-authored-by: Robert Scott <code@humanleg.org.uk>
Co-authored-by: Robert Hensing <roberth@users.noreply.github.com>
Co-authored-by: Dylan Baker <dylan@pnwbakers.com>
Co-authored-by: FliegendeWurst <2012gdwu+github@posteo.de>
Co-authored-by: Fabian Affolter <fabian@affolter-engineering.ch>
Co-authored-by: Trolli Schmittlauch <t.schmittlauch+nixos@orlives.de>
Co-authored-by: Benjamin Hipple <bhipple@protonmail.com>
Co-authored-by: Samuel Dionne-Riel <samuel@dionne-riel.com>
Co-authored-by: figsoda <figsoda@pm.me>
Co-authored-by: Matthieu Coudron <teto@users.noreply.github.com>
Co-authored-by: Aaron Andersen <aaron@fosslib.net>
Co-authored-by: Silvan Mosberger <contact@infinisil.com>
Co-authored-by: kat witch <kat@kittywit.ch>
Co-authored-by: Mario Rodas <marsam@users.noreply.github.com>
Co-authored-by: Mauricio Collares <mauricio@collares.org>
Co-authored-by: Reed <mrnitsuj1@gmail.com>
Co-authored-by: happysalada <raphael@megzari.com>
Co-authored-by: Matt Votava <mvnetbiz@gmail.com>
Co-authored-by: Jan Tojnar <jtojnar@gmail.com>
Co-authored-by: Robert Schütz <dev@schuetz-co.de>
Co-authored-by: Vladimír Čunát <v@cunat.cz>
Co-authored-by: ajs124 <git@ajs124.de>
Co-authored-by: Сухарик <65870+suhr@users.noreply.github.com>
Co-authored-by: Paul-Henri Froidmont <github.contact-s813yn@froidmont.org>
Co-authored-by: lasers <lasers@users.noreply.github.com>
Co-authored-by: Patrick Hilhorst <git@hilhorst.be>
Co-authored-by: Git Machete Release Bot <gitmachete@virtuslab.com>
Co-authored-by: Doron Behar <doron.behar@gmail.com>
Co-authored-by: Izorkin <izorkin@elven.pw>
Co-authored-by: Daniël de Kok <me@danieldk.eu>
Co-authored-by: Thiago Franco de Moraes <totonixsame@gmail.com>
Co-authored-by: Akshay Mankar <itsakshaymankar@gmail.com>
Co-authored-by: endorama <endorama@users.noreply.github.com>
Co-authored-by: Jack K <dev@j-k.io>
Co-authored-by: kfollesdal <kristoffer.follesdal@bkk.no>
Co-authored-by: Florian Klink <flokli@flokli.de>
Co-authored-by: Cole Helbling <cole.e.helbling@outlook.com>
Co-authored-by: Pascal Bach <pascal.bach@nextrem.ch>
Co-authored-by: Thomas Tuegel <ttuegel@mailbox.org>
Co-authored-by: zowoq <59103226+zowoq@users.noreply.github.com>
Co-authored-by: Bernardo Meurer <bernardo@meurer.org>
Co-authored-by: Diego Rodriguez <diego@noteable.io>
Co-authored-by: kvtb <76634406+kvtb@users.noreply.github.com>
Co-authored-by: Robin Gloster <mail@glob.in>
Co-authored-by: Léo Gaspard <leo@gaspard.io>
